package.json

```diff
@@ -67,7 +67,7 @@
     "first-run": "~2.0.0",
     "graphql-tag": "~2.10.1",
     "identity-obj-proxy": "~3.0.0",
-    "jest": "~24.3.1",
+    "jest": "~25.3.0",
```

Comment:
The test helpers were not working well because of some quirky behavior in Jest 24. Upgrading it was very smooth, and added a lot of robustness. Only a few changes to test implementations were necessary. I think we should jump at the chance.